src/layouts/CoreLayout/CoreLayout.scss

```diff
@@ -1,2 +1,11 @@
 .mainContainer {
 }
+section:first-child {
```

Comment:
Oh... Whoops. Yeah, lets keep these to actual classes that we can reference by name instead of referencing the element